### PR TITLE
Change the starboard embed author name to their displayname.

### PR DESCRIPTION
**Pull request by GianniKoch**

### CODE REVIEW

File: source/POI.DiscordDotNet/POI.DiscordDotNet.csproj [hunk 1]
```diff
@@ -9,11 +9,10 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="DSharpPlus" Version="4.4.2" />
-        <PackageReference Include="DSharpPlus.CommandsNext" Version="4.4.2" />
-        <PackageReference Include="DSharpPlus.Interactivity" Version="4.4.2" />
-        <PackageReference Include="DSharpPlus.Rest" Version="4.4.2" />
-        <PackageReference Include="DSharpPlus.SlashCommands" Version="4.4.2" />
+        <PackageReference Include="DSharpPlus" Version="5.0.0-nightly-01663" />
```
Comment (grYs51) [inline source/POI.DiscordDotNet/POI.DiscordDotNet.csproj:12]: Did you check all major changes?

Comment (ErisApps) [reply source/POI.DiscordDotNet/POI.DiscordDotNet.csproj:12]: Testing in production.

**Review by ErisApps — APPROVED** [review]

File: source/POI.DiscordDotNet/POI.DiscordDotNet.csproj (hunk 1 above)
Comment (GianniKoch) [reply source/POI.DiscordDotNet/POI.DiscordDotNet.csproj:12]: > Testing in production.

afbeelding

**Pull request merged**